scripts/uitest.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+
+# Define paths to uitest directories and command.
+script_dir=`dirname $0`
+root_dir=`realpath $script_dir/../`
+uitest_dir=$root_dir/uitest
+uitest=$uitest_dir/devenv/bin/uitest
+
+# Define default parameters.
+cloud="lxd"
+archive=`ls $root_dir/dist/jujugui-*.tar.bz2`
+args=""
+
+# Parse the arguments.
+while [[ $# > 0 ]]; do
+key="$1"
+case $key in
+    -c|--cloud)
+    cloud="$2"
+    shift
+    ;;
+    --gui-archive)
+    archive="$2"
+    shift
+    ;;
+    *)
+    args="$args $key"
+    ;;
+esac
```

Comment:
--help override all other options allrady, so I consider it ok to just propagate that without any special handling.